spec/std/dir_spec.cr

```diff
@@ -1,10 +1,5 @@
 require "spec"
 
-private def assert_dir_glob(expected_result, *patterns)
```

Comment:
If find it impractical as it doesn't transmit the location of the failing expectation (all failing specs fail in line 5). This might not be that important because specs should generally match (except when refactoring or improving that code).

It also doesn't do much - essentially it just adds the two `sort`s - and feels like if anything it is hiding functionality.